iso15118/secc/states/din_spec_states.py

```diff
@@ -794,28 +780,12 @@ async def process_message(
 
         current_demand_req: CurrentDemandReq = msg.body.current_demand_req
 
-        self.comm_session.evse_controller.ev_data_context.ev_session_context.soc = (
-            current_demand_req.dc_ev_status.ev_ress_soc
-        )
-
-        self.comm_session.evse_controller.ev_data_context.ev_session_context.remaining_time_to_bulk_soc_s = (  # noqa: E501
-            None
-            if current_demand_req.remaining_time_to_bulk_soc is None
-            else current_demand_req.remaining_time_to_bulk_soc.get_decimal_value()
-        )
-
-        self.comm_session.evse_controller.ev_data_context.ev_session_context.remaining_time_to_full_soc_s = (  # noqa: E501
-            None
-            if current_demand_req.remaining_time_to_full_soc is None
-            else current_demand_req.remaining_time_to_full_soc.get_decimal_value()
-        )
-
-        self.comm_session.evse_controller.ev_charge_params_limits.ev_max_current = (
-            current_demand_req.ev_max_current_limit
-        )
+        self.comm_session.evse_controller.ev_data_context.update_charge_loop(current_demand_req)
```

Comment:
```suggestion
        self.comm_session.evse_controller.ev_data_context.update_charge_loop_parameters(
            current_demand_req
        )
```

Reply:
it was updated, closing for a fresh review